config/testV5/config.js

```diff
@@ -154,6 +154,23 @@ module.exports = {
 		{title: "Cookies", type: "haveRead", page: "cookies", fileName: "cookies.html"}
 	],
 	userNotice: false,
-	termsUpdatedAt: 12345678
+	termsUpdatedAt: 12345678,
+
+	mail: {
```

Comment:
i dont' think you need to put in actual mailer info?

your unit test is mocking the mailer

e2e may fail, but it returns ok to the user anyway?

Reply:
if you have to, put in some super dummy data with https://ethereal.email/

don't spam google smtp :P

Reply:
I think we said that if an email fails to be sent we should let the user know

Reply:
This is the config for v5 tests, has nothing to do with actual processing 

Yes you need mailer configured when you run it, Just wondering if we need to set something for the test (i.e. does it fail in the wrong way if it does? if the answer is yes, use etheral to generate dummy but valid credentials)

Reply:
Just generate the ethereal mail on the fly. nodemailer has an api for that - so it changes everytime and  no one can log in and snoop on the emails.

They have an example on the link provided. 
![image](https://user-images.githubusercontent.com/11945337/158595533-db0a3b4e-78e5-45a5-bbef-a6e7e3e0713b.png)